presto-spi/src/main/java/com/facebook/presto/spi/function/SqlInvokedFunction.java

```diff
@@ -52,6 +53,7 @@ public class SqlInvokedFunction
     private final SqlFunctionId functionId;
     private final FunctionVersion functionVersion;
     private final Optional<SqlFunctionHandle> functionHandle;
+    private final ComplexTypeFunctionDescriptor descriptor;
```

Comment:
This descriptor is never used, did you plan to use it via
```
@Override
    public ComplexTypeFunctionDescriptor getComplexTypeFunctionDescriptor()
    {
        return descriptor;
    }
```

Reply:
Yes, it is used through getComplexTypeFunctionDescriptor() in order to build FunctionMetadata object in [BuiltInTypeAndFunctionNamespaceManager](https://github.com/prestodb/presto/blob/master/presto-main/src/main/java/com/facebook/presto/metadata/BuiltInTypeAndFunctionNamespaceManager.java#L1117)